core/src/env/test_env.rs

```diff
@@ -146,9 +146,9 @@ impl TestEnvData {
     /// Resets `self` as if no contract execution happened so far.
     pub fn reset(&mut self) {
         self.storage.clear();
-        self.caller.clear();
+        self.caller = srml::Address([0; 32]);
```

Comment:
Can you please use `0x0` here instead of `0`?
It is just to signal that this is a byte blob.